source/content/Codecamps.md

```diff
@@ -0,0 +1,20 @@
+Code Camps
+==========
+
+
+```{toctree}
+:maxdepth: 2
+:caption: Contents:

```

Comment:
Consider using [`:glob:`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-option-toctree-glob) here and gather all current and future `CodeCamp*.md` entries together.  Assume same for `NIAC*.md`.

Reply:
thanks Pete, took your suggestion and tried it but was unable to make :glob: work regardless how I declared it in the markdown, it just would not match to the files resulting in "_WARNING: document isn't included in any toctree_" messages during build.